cypress/integration/playground/application-shell.js

```diff
@@ -1,51 +1,75 @@
-/* eslint-disable jest/valid-expect-in-promise */
 import { encode } from 'qss';
 import { LOGOUT_REASONS } from '@commercetools-frontend/constants';
-import { URL_BASE, URL_STATE_MACHINES } from '../../support/urls';
+import {
+  URL_BASE,
+  URL_STATE_MACHINES,
+  ENTRY_POINT_STATE_MACHINES,
+} from '../../support/urls';
 
 describe('when user is authenticated', () => {
+  beforeEach(() => {
+    cy.loginByOidc({ entryPointUriPath: ENTRY_POINT_STATE_MACHINES });
+  });
   it('should log out with reason "user"', () => {
-    cy.login({ redirectToUri: URL_STATE_MACHINES });
-
     cy.findByRole('button', { name: /open user settings menu/i }).click();
-    cy.findByRole('link', { name: /logout/i }).click();
```

Comment:
We don't have a logout page anymore, so we simply check that the link is correct (as it redirects to another domain)